pom.xml

```diff
@@ -152,13 +132,10 @@
   <build>
     <plugins>
       <plugin>
-        <!-- critical because sshd needs to be loaded first, otherwise will just bail out -->
-        <!-- https://www.jenkins.io/doc/developer/plugin-development/dependencies-and-class-loading/ -->
         <groupId>org.jenkins-ci.tools</groupId>
         <artifactId>maven-hpi-plugin</artifactId>
         <extensions>true</extensions>
         <configuration>
-          <pluginFirstClassLoader>true</pluginFirstClassLoader>
           <compatibleSinceVersion>1.5</compatibleSinceVersion>
```

Comment:
Probably should have had this updated?

Reply:
we can for the next release. from https://github.com/jenkinsci/ssh-agent-plugin/pull/48#issuecomment-805759155, it shouldn't matter for `1.22` no?

Reply:
IIUC `compatibleSinceVersion` is only relevant for settings changes, and there are none here.

Reply:
Doesn't this break existing configurations that rely on this feautre? It's not _exactly_ configuration compatibility in the strict sense, but I don't think anyone whose jobs we break will care about that distinction.

Reply:
Whatever you think appropriate.

Reply:
If we have an agreement here, I need to cut a release really soon because I already published `1.22`